public/icons/down.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="8" height="12" viewBox="0 0 8 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
import ImportExportIcon from '@material-ui/icons/ImportExport';

-> Mozna użyc ikonki powyzszej z material-ui ale jak wolisz :)

Reply:
Też się zastanawiałam, ale nie wiem do końca, jak to sortowanie będzie działać (możemy przypuszczać, no ale nie wiemy do końca), więc wolałam rozbić to na dwie oddzielne ikonki, bo ta w/w to jest jedna. A w samym materialu nie było dwóch oddzielnych góra/dół wystarczająco ładnych ;)